assets/reducers/report.js

```diff
@@ -0,0 +1,19 @@
+import { REQUEST_REPORT, RECEIVE_REPORT } from '../actions/report'
+import { morph } from '../utils'
+
+const defaultState = {
+    name: null
```

Comment:
It's used after returning the blob from a successful request. `getState().report.name`.